### PR TITLE
rdbtools: init at 0.1.14

### PR DESCRIPTION
**Pull request by offlinehacker**
###### Motivation for this change

Package rdbtools

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by offlinehacker** [comment]
Fixed eval error, should be good now

File: pkgs/development/tools/rdbtools/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{ stdenv, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
```
Comment (jonringer) [inline pkgs/development/tools/rdbtools/default.nix]:
```suggestion
python.pkgs.buildPythonApplication rec {
```

File: pkgs/development/tools/rdbtools/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  pname = "rdbtools";
+  version = "0.1.14";
+
+  src = python3Packages.fetchPypi {
```
Comment (jonringer) [inline pkgs/development/tools/rdbtools/default.nix]:
```suggestion
  src = python.pkgsfetchPypi {
```

File: pkgs/development/tools/rdbtools/default.nix [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    inherit pname version;
+    sha256 = "03vdwwkqz8py6c3wfgx402rn8pjjfn44w3gbxzr60lbkx27m63yj";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ redis python-lzf ];
```
Comment (jonringer) [inline pkgs/development/tools/rdbtools/default.nix]:
```suggestion
  propagatedBuildInputs = with python.pkgs; [ redis python-lzf ];
```

File: pkgs/top-level/all-packages.nix [hunk 4]
```diff
@@ -5949,6 +5949,8 @@ in
 
   rc = callPackage ../shells/rc { };
 
+  rdbtools = callPackage ../development/tools/rdbtools { };
```
Comment (jonringer) [inline pkgs/top-level/all-packages.nix]: set python version here
```suggestion
  rdbtools = callPackage ../development/tools/rdbtools {
    python = python3;
  };
```

**Review by jonringer — COMMENTED** [review]
probably future proofing, but only referencing the python interpreter allows better flexibility with overrides

**Comment by offlinehacker** [comment]
Ok, i implemented suggested changes. Also using default version of `python` works just fine, so no need to pass `python3`.

**Comment by jonringer** [comment]
i would highly recommend python3, python2 is EOL in 2 weeks

**Comment by offlinehacker** [comment]
Ok, but works both with both versions, so when we change the default version to `python3` it should work without any change. I was just thinking to leave it as is, so you do need `python3` on system if only using this package.

File: pkgs/top-level/all-packages.nix (hunk 4 above)
Comment (jonringer) [reply pkgs/top-level/all-packages.nix]: python2 is near EOL, use python3 please. I know im going to have to revisit this in a few weeks if i dont

**Comment by offlinehacker** [comment]
Ok, i set `python3` explicitly, should be ready to merge now.

**Comment by jonringer** [comment]
> Ok, but works both with both versions, so when we change the default version to `python3` it should work without any change. I was just thinking to leave it as is, so you do need `python3` on system if only using this package.

It would be the same thing with python2, they python closure in nixpkgs is ~110MB, and it get's "reused" by many other packages, so it's actual cost is quite amortized.

**Review by jonringer — APPROVED** [review]
diff LGTM
commits LGTM

```
[7 built, 0.0 MiB DL]
https://github.com/NixOS/nixpkgs/pull/68631
4 package were built:
python27Packages.python-lzf python37Packages.python-lzf python38Packages.python-lzf rdbtools
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build python27Packages.python-lzf python37Packages.python-lzf python38Packages.python-lzf rdbtools

**Pull request merged**